europe/ISL

```diff
@@ -0,0 +1,312 @@
+#------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+#
+#    PROJ implementations of current and legacy Icelandic coordinate systems
+#
+#------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+#
+#  This file is Work-In-Progress, and should not be used without testing. It is
+#  the official definition and implementation of transformations between different Iceland's systems.
```

Comment:
This is self-contradictory. These are the official transformations but I shouldn't trust them? @dalia2, you and Gudmundur have done a solid job on these transformations. You should stand by them without any precautions.

Reply:
I have edited the file regarding that.